packages/browser/README.md

```diff
@@ -6,6 +6,10 @@ The main documentation is at the [root of the repository](../../README.md).
 ## Underlying libraries
 
 `solid-client-authn-browser` is based on [`oidc-client-js`](https://github.com/IdentityModel/oidc-client-js). 
-However, the latter lacks some features that are necessary to provide the developer
-experience we specifically want for the Solid ecosystem, so we developed  
-[`oidc-dpop-client-browser`](https://www.npmjs.com/package/@inrupt/oidc-dpop-client-browser) to add these features.
+This mean a standard, certified library is used when signing in to a Pod.
+
+However, `oidc-client-js` lacks some features that are necessary to provide the
+developer experience we specifically want for the Solid ecosystem, so we
+developed  
+[`oidc-dpop-client-browser`](https://www.npmjs.com/package/@inrupt/oidc-dpop-client-browser) 
```

Comment:
I don't think we want to advertise this library; we don't want to be an upstream for people using this, but rather have it integrated in oidc-client-js proper.